vulnerabilities/importers/project_kb_msr2019.py

```diff
@@ -80,15 +68,23 @@ def fetch(self):
     @staticmethod
     def to_advisories(csv_reader):
         # Project KB MSR csv file has no header row
-        advsiories = []
+        advisories = []
         for row in csv_reader:
             vuln_id, proj_home, fix_commit, _ = row
             commit_link = proj_home + "/commit/" + fix_commit
-            advsiories.append(
+
+            if re.compile(r"CVE-\d+-\d+").match(vuln_id):
+                reference = Reference(url=commit_link)
+
+            else:
+                reference = Reference(url=commit_link, reference_id=vuln_id)
+                vuln_id = ""
```

Comment:
Is this needed?

Reply:
YES . Either the data makes vuln_id some cve or some apache collections id. In the latter case we instead of assigning apache collections id as a vulnerability id we throw it as a reference.